bciers/apps/reporting/src/app/components/attachments/AttachmentsPage.tsx

```diff
@@ -32,7 +36,7 @@ const AttachmentsPage: React.FC<HasReportVersion> = async ({ version_id }) => {
       version_id={version_id}
       initialUploadedAttachments={uploadedAttachmentsDict}
       navigationInformation={navInfo}
-      isVerificationStatementMandatory={needsVerification}
+      isVerificationStatementMandatory={isEIO ? false : needsVerification}
```

Comment:
We should add that logic in the backend in the `getReportNeedsVerification(version_id)` call, not here
Otherwise:
- we have an extra place (here) where we implement additional logic that needs to be tested
- the getReportNeedsVerification method returns something inaccurate and doesn't take in account EIOs, for other consumers